docs/testplan/token-payment-processor-testplan.md

```diff
@@ -0,0 +1,31 @@
+# Token Payment Processor Testplan
+
+&nbsp;
+
+## Background
+This testplan is going to cover the token payment processor, it will go over must haves and additional features that we can add on later. 
+
+UI relating to token payment processing can be seen on [Billing Page](https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI-Public?node-id=11080%3A68109) figma design.
+
+&nbsp;
+
+&nbsp;
+
+<google-sheets-html-origin><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+
+| Test Scenario |               Test Case              |                                                                                                                                                Description                                                                                                                                               |                                                                        Comments                                                                        |
+|:-------------:|:------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| Transaction   | Block Chain Reorg                    | If there is a naturally occurring chain reorg then past transactions in the old chain are deactivated due to a chain reorganization (this should be visible in UI, pending to cancelled)                                                                                                                 | Must have early on (Alpha)                                                                                                                             |
```

Comment:
I'm not sure if we are actually going to show these transactions in the UI at all.

- If the reorg happens before we are aware of the new transaction then we won't know about that transaction at all. 
- If the reorg is less than the number of confirmations we require it will be deleted from our database.
- if the reorg is larger than the number of confirmations we require our database could transactions from different blocks with the same block number.